app/assets/v2/js/grants/fund.js

```diff
@@ -60,10 +60,10 @@ $(document).ready(function() {
 
         $('#gas_price').val(realGasPrice);
 
-        let realApproval = Number(((data.approve * 10 ** decimals) + realGasPrice) * data.amount_per_period);
-
         let realTokenAmount = Number(data.amount_per_period * 10 ** decimals);
 
+        let realApproval = Number((realTokenAmount + realGasPrice) * data.approve);
```

Comment:
shouldnt this be (amount_in_tokens_per_period + gas_price_per_period) * num_periods ?

Reply:
@owocki that is what it is. realApproval = (converted_token_amount + converted_gas_price_per_period) * num_periods. I can change a variable name or two to make it more little more readable.